src/main/scala/molmed/qscripts/DNABestPracticeVariantCalling.scala

```diff
@@ -150,6 +150,12 @@ class DNABestPracticeVariantCalling extends QScript
   @Argument(doc = "Emit the original qualities in the BQSR step", fullName = "emit_original_quals", shortName = "orgQuals", required = false)
   var emitOriginalQuals: Boolean = false
 
+  @Argument(doc = "use flag --skip-duplicated with qualimap, i.e skip alignments from analysis. Three modes: " +
```

Comment:
I think that this should be provided with a sensible default, e.g. whatever we are planing to use in the production pipeline.